addon/components/grid-stack.js

```diff
@@ -122,22 +121,20 @@ export default class GridStackComponent extends Component {
     // we must manually enable it
     if (!(this.options && this.options.staticGrid)) {
       let grid = this.gridStack;
-      let itemClass = grid.opts.itemClass;
-      jQuery(this.element)
-        .children(`.${itemClass}`)
-        .each((_, el) => {
-          // only enable items that are supposed to mobile
-          let noMove = el.getAttribute('data-gs-no-move');
-          let noResize = el.getAttribute('data-gs-no-resize');
-
-          if (!noMove) {
-            grid.movable(el, true);
-          }
-
-          if (!noResize) {
-            grid.resizable(el, true);
-          }
-        });
+
```

Comment:
didn't scan the code too carefully, but if 'data-gs-no-move' is missing, why do you need to call movable(true) ? isn't the grid already doing the right thing ?